Cargo.lock

```diff
@@ -0,0 +1,182 @@
+[[package]]
+name = "bit-vec"
```

Comment:
This repo wll not be updated frequently after development.
So, I think `Cargo.lock` is helpful to keep a available dependency tree.
If there is no `Cargo.lock`, maybe two month later, it can not be compiled successfully.

Keep or remove, both are ok, IMHO.
I will remove the commit "Add Cargo.lock file.", after rebase.

Reply:
https://github.com/rust-lang/cargo/issues/315